core/autocrypt/mime.py

```diff
@@ -88,6 +91,7 @@ def parse_ac_headervalue(value):
         if name == "key":
             value = "".join(value.split())
         result_dict[name] = value
+        logger.debug('len(result_dict) %s', len(result_dict))
     return result_dict
 
```

Comment:
also here (and in other places) please remove logger.debug calls -- apart from pdb.set_trace() you can also just use "print()" commands which will be shown for failing tests in a special section.